pipeline_steps/irr_role_tests.groovy

```diff
@@ -0,0 +1,72 @@
+def run_irr_tests() {
+  common.conditionalStage(
+    stage_name: "Run IRR Tests",
+    stage: {
+      pubcloud.runonpubcloud {
+        try {
+          ansiColor('xterm') {
+            dir("${env.WORKSPACE}/${env.ghprbGhRepository}") {
+              print("Triggered by PR: ${env.ghprbPullLink}")
+              checkout([$class: 'GitSCM',
+                branches: [[name: "origin/pr/${env.ghprbPullId}/merge"]],
```

Comment:
I don't intend this to work with a periodic job at this point. only PRs for now.